test/parallel/test-inspect-open.js

```diff
@@ -0,0 +1,113 @@
+// --inspect-port to child process is ignored :-(
+'use strict';
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
+const path = require('path');
+const url = require('url');
+
+const common = require('../common');
```

Comment:
yes, sorry, I copied the test from an existing one and didn't notice it didn't fit the current style. I'll fix this.